ProjFS.Mac/PrjFSLib/prjfs-log/prjfs-log.cpp

```diff
@@ -145,7 +145,7 @@ static const char* KextLogLevelAsString(KextLog_Level level)
         return "Error";
     case KEXTLOG_INFO:
         return "Info";
-    case KEXTLOG_NOTE:
+    case KEXTLOG_DEFAULT:
         return "Note";
```

Comment:
Should we change "Note" to "Default" as well?

Reply:
Of course, thanks for catching that. Fixed in the new version.